SequenceDividedOnClasses/AnalizeSequence.java

```diff
@@ -0,0 +1,26 @@
+/**  
+ * This class analize your sequence if it is non-decreasing.
+ */
+public class AnalizeSequence {
+
+  /**
+   * This method determines if the sequence is non-decreasing or not
+   * and puts the answer to the console.
+   *
+   * @param sequenceStringDivided checking sequence
+   * @exception NumberFormatException if e puts not integer numbers 
+   */
+  public void determineSequence(String[] sequenceStringDivided) {
+    try {
+      for (int i = 0; i + 1 < sequenceStringDivided.length ; i++) {
+        if (Integer.parseInt(sequenceStringDivided[i]) > Integer.parseInt(sequenceStringDivided[i + 1])) {
+          System.out.println("Your sequence is not non-decreasing");

```

Comment:
Single responsibility principle violation - method must only check sequence (1 responsibility), not "check, verify and ouptut results" (3 responsibilities)